pkg/applications/helmclient/client_test.go

```diff
@@ -0,0 +1,411 @@
+/*
+Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helmclient
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/distribution/distribution/v3/configuration"
+	ociregistry "github.com/distribution/distribution/v3/registry"
+	"github.com/phayes/freeport"
+	"helm.sh/helm/v3/pkg/chart"
+	"helm.sh/helm/v3/pkg/chartutil"
+	"helm.sh/helm/v3/pkg/cli"
+	"helm.sh/helm/v3/pkg/provenance"
+	"helm.sh/helm/v3/pkg/pusher"
+	helmregistry "helm.sh/helm/v3/pkg/registry"
+	"helm.sh/helm/v3/pkg/repo/repotest"
+	"helm.sh/helm/v3/pkg/uploader"
+
+	kubermaticlog "k8c.io/kubermatic/v2/pkg/log"
+
+	cmdtesting "k8s.io/kubectl/pkg/cmd/testing"
+	"sigs.k8s.io/yaml"
+)
+
+const defaultNs = "default"
+
+func TestHelmClient_NewShouldFailWhenRESTClientGetterNamespaceIsDifferentThanTargetNamespace(t *testing.T) {
```

Comment:
I am not sure about the func names throughout this file: I saw that we rarely use underscore prefixes in our code for test funcs.

Given that it is a small package, I think It is fine to drop the `HelmClient` part of the prefix

Reply:
done :)